app/src/main/java/com/sunasterisk/loltournaments/data/source/remote/ApiService.kt

```diff
@@ -23,7 +23,7 @@ interface ApiService {
         @Query(QUERY_FILTER_ID) tournamentId: Int
     ): List<Tournament>
 
-    @GET(PATH_MATCH)
+    @GET("$PATH_MATCH?$QUERY_SEARCH_PER_PAGE_100")
     suspend fun getMatchByTournamentId(
         @Query(QUERY_FILTER_TOURNAMENT_ID) matchId: Int
     ): List<Match>
```

Comment:
Sao lại thêm query như thế em ? 
```suggestion
    @GET(PATH_MATCH)
    suspend fun getMatchByTournamentId(
        @Query(QUERY_FILTER_TOURNAMENT_ID) matchId: Int, 
        @Query("per_page") perPage: Int = 100
    ): List<Match>
```